README.md

```diff
@@ -12,6 +13,17 @@
 
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
+#### [Github Documentation](https://docs.github.com/en/github)
+### [Git and Github tutorials on Codecademy](https://www.codecademy.com/learn/learn-git)
```

Comment:
Why does this link have a different header than the other links?

Reply:
Hi Michael, That happened by mistake, I just fixed. All has to be the same. Thanks for that